__tests__/integration/services/applications.js

```diff
@@ -1,12 +1,118 @@
 const applicationService = require('../../../src/services/applications')
+const { Applications } = require('../../../src/database')
+
+const APP_ID = 'create_service_1'
 
 describe('Application service', () => {
-  describe.skip('create',() => {
-    it('creating with valid payload', async () => {
-      const response = await applicationService.create()
+  beforeEach(() => {
+    return Applications.delete(APP_ID).catch()
+  })
+
+  describe('create',() => {
+    it('creating using only app_id', async () => {
+      const response = await applicationService.create({
+        app_id: APP_ID,
+      })
+
+      expect(response).toHaveProperty('app_id', APP_ID)
+      expect(response).toHaveProperty('created_at')
+      expect(response).toHaveProperty('updated_at')
+      expect(response).toHaveProperty('status', 'active')
+      expect(response).toHaveProperty('api_key')
+    })
+
+    it('creating using app_id and secret', async () => {
+      const response = await applicationService.create({
+        app_id: APP_ID,
+        secret: 'super.secret.key'
+      })
+
+      expect(response).toHaveProperty('app_id', APP_ID)
+      expect(response).toHaveProperty('secret', 'super.secret.key')
+      expect(response).toHaveProperty('created_at')
+      expect(response).toHaveProperty('updated_at')
+      expect(response).toHaveProperty('status', 'active')
+      expect(response).toHaveProperty('api_key')
+    })
+
+    it('creating using app_id and status', async () => {
+      try {
+        await applicationService.create({
+          app_id: APP_ID,
+          status: 'super_power_status',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
          status: "super_power_status",
```